CONTRIBUTING.md

```diff
@@ -223,14 +238,27 @@ For more information about those scripts, see their documentation.
 - [References](http://doc.qt.io/qt-5/stylesheet.html)
 
 Create good palettes with these tools. For example, on paletton, choose
-  three colors from greyish light (foreground), greyish dark (background)
-  and three more colorfull colors (selection). Greyish colors have a litle
-  bit of the main color, so it is nice to change it if you change the main
-  color.
+three colors from greyish light (foreground), greyish dark (background)
+and three more colorfull colors (selection). Greyish colors have a litle
+bit of the main color, so it is nice to change it if you change the main
+color.
 
 - [Paletton.com](http://paletton.com/)
 - [Coolors.co](https://coolors.co/)
 
+As a minimal guide to create new icons (svg) images, we list two main sources.
+
+ - [Material](https://material.io/design/iconography/product-icons.html#grid-keyline-shapes)
+ - [KDE](https://hig.kde.org/style/icon.html)
+
+Main characteristics are of SVG images are:
+
+- Size: 32px X 32px;
+- Border: 2px border space;
+- Line: 2px minimum thickness. Complementary 4px and 8px thickness;
```

Comment:
I would add also that these lines and shape should align to the grid as much as possible (depending on the shape of course)

Reply:
Maybe adding an image?

https://hig.kde.org/style/icon.html
<img width="534" alt="Screen Shot 2019-05-24 at 17 10 50" src="https://user-images.githubusercontent.com/3627835/58358995-eec1ce80-7e46-11e9-9358-31a3f4efe87b.png">

Reply:
Oh, yes, excellent! I will add.